q2_types/feature_table/_transformer.py

```diff
@@ -12,26 +12,36 @@
 import pandas as pd
 import qiime
 
-from . import BIOMV1Format
+from . import BIOMV1Format, BIOMV210Format
```

Comment:
Just discussed this with @ebolyen and @jairideout. Since we are renaming the dir fmts here, can you also rename `BIOMV1Format` -> `BIOMV100Format`, to match your new `BIOMV210Format`? I know thats a little outside of your original PR but it would be convenient to knock it all out at once.